### PR TITLE
color_changed in add client and search button

### PR DESCRIPTION
**Pull request by Tushar403**
Targets #1686 

### Description
changed background color of both add client and search buttons similar to the active clients.

### Checklist:
 I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Client/Resources/views/index.blade.php [hunk 1]
```diff
@@ -58,10 +58,14 @@
 
                 @empty
                     <tr>
-                        <td colspan="2">
+                        <td>  </td>
+                        <td> - </td>
+                        <td>  </td>
```
Comment (P4NK4J) [inline Modules/Client/Resources/views/index.blade.php:63]: Why you are removing colspan and adding manual rows ?

File: Modules/Client/Resources/views/index.blade.php [hunk 2]
```diff
@@ -58,10 +58,14 @@
...
                             <p class="my-4 text-left">No {{ config('client.status')[request()->input('status', 'active')] }} clients found.</p>
                         <td>
                     </tr>
+                   
+
```
Comment (rathorevaibhav) [inline Modules/Client/Resources/views/index.blade.php:68]: please remove extra lines.

**Review by P4NK4J — COMMENTED** [review]
left a comment, please check

File: Modules/Client/Resources/views/index.blade.php [hunk 3]
```diff
@@ -12,7 +12,7 @@
     <div class="d-none d-md-flex justify-content-between my-2">
         @include('client::menu_header')
         @can('clients.create')
-            <a href= "{{ route('client.create') }}" class="btn btn-info text-white">Add client</a>
+            <a href= "{{ route('client.create') }}" class="btn btn-info text-white active">Add client</a>
```
Comment (rathorevaibhav) [inline Modules/Client/Resources/views/index.blade.php:15]: what does `active` class do? Why have you added it here?

File: Modules/Client/Resources/views/index.blade.php (hunk 1 above)
Comment (rathorevaibhav) [inline Modules/Client/Resources/views/index.blade.php:63]: can you explain these changes? I don't understand the reason for these changes from the issue description.

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: Modules/Client/Resources/views/index.blade.php (hunk 1 above)
Comment (Tushar403) [reply Modules/Client/Resources/views/index.blade.php:63]: because i got confused while performig a task, i thought i have to make table also.

Comment (Tushar403) [reply Modules/Client/Resources/views/index.blade.php:63]: i fixed this issue in master branch

**Pull request closed without merging**